src/main/java/com/google/cloud/tools/gradle/endpoints/framework/client/EndpointsClientExtension.java

```diff
@@ -30,16 +30,20 @@
   private File genSrcDir;
   private final File clientLibDir;
   private final File genDiscoveryDocsDir;
+  private final File genOpenApiDocsDir;
```

Comment:
It does not appear likely that we need to modify the client plugin. The client plugin is designed for generating client libraries for client consumers of endpoints v1.1.

For OpenApi specs, I believe you're going to delegate to another tool. Are you using something to do that? And how are you using it?